src/patternfly/components/DualListSelector/dual-list-selector.scss

```diff
@@ -339,7 +340,7 @@ $pf-v6-c-dual-list-selector__item--MaxNesting: 10;
   min-width: var(--#{$dual-list-selector}__item-toggle-icon--MinWidth);
   color: var(--#{$dual-list-selector}__item-toggle-icon--Color, inherit);
   text-align: center;
-  transition: transform var(--#{$dual-list-selector}__item-toggle-icon--TransitionDuration) var(--#{$dual-list-selector}__item-toggle-icon--TransitionTimingFunction);
+  transition: var(--#{$dual-list-selector}__item-toggle-icon--Transition); // TODO remove shorthand in breaking change
```

Comment:
This one can be longhand, too, and you can remove `--#{$dual-list-selector}__item-toggle-icon--Transition`